README.md

```diff
@@ -3,7 +3,29 @@ force-torque-sensor
 
 A catkin package to read measurements from any ATI FT sensor via XML
 
-ToDO:
- - [ ] Use force-torque-sensor controller from ros_control
+Done:
+ - [x] Use force-torque-sensor controller from ros_control
  - [x] Create an URDF of the sensor to visualize the forces and frames
- - [ ] Create plugin to be used in simulation
+ - [x] Create plugin to be used in simulation
+ - [x] Merge nano17 and gamma sensor 
+
+Use in simulation:
+
+1) on first terminal for nano17 ft sensor type:
+
+roslaunch force_torque_sensor ft_simulation_display.launch use_gamma:=false 
```

Comment:
Is there a way to pass a string instead of a true/false arg?

Would be nice to have something like:

`roslaunch force_torque_sensor ft_simulation_display.launch use_model:=gamma`